src/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface Category {
+    id: number;
+    title: string;
+}
+
+export interface Question {
+    id: number;
+    title: string;
+    body: string;
+    format: string;
+    releaseDate: string;
+    expirationDate: string;
+    difficulty: string;
+    categories: Category[];
```

Comment:
would we need the img source and perhaps an alternate description too for the graphs?

Reply:
Yes, we will be adding in more properties as they are added in the backend. Currently the backend doesn't have an image_url yet, so it isn't yet reflected in the type.